contracts/Commitment/Commitment.sol

```diff
@@ -425,6 +426,8 @@ contract Commitment is
             ETHER_LOCK.controllerSucceeded();
             EURO_LOCK.controllerSucceeded();
         }
+        // burn Neumarks after state change to prevent theoretical re-entry
+        NEUMARK.burn(nmkToBurn);
```

Comment:
As in the other cases, you probably don't want to trigger the burning if there is nothing to burn (`nmkToBurn == 0`).

Reply:
in this case I'd like to have a trace that there was attempt to burn 0 Ulps of Neumark. This means that whole whitelist was realized by investors. This is actually useful information. I've checked. Logs will be generated.

Reply:
Ok, but you will get a similar log also for transition to Whitelist state.